### PR TITLE
blueline/crosshatch API-28 config

### LINKED ISSUE #135: crosshatch (Pixel 3 XL) support
I don't have a device yet, so haven't played around with building an bootable image yet: https://developers.google.com/android/images#crosshatch. Figured it would be good to have an open item to track any progress though.

### LINKED ISSUE #136: blueline (Pixel 3) support
I don't have a device yet, so haven't played around with building an bootable image yet: https://developers.google.com/android/images#blueline. Figured it would be good to have an open item to track any progress though.

### PR DESCRIPTION
**Pull request by cernekee**
blueline was tested with rattlesnakeos on real hardware.

crosshatch is untested (but looking at PD1A.180720.030, the file list on /system is identical between these devices except for init.$DEVICE.rc).

closes #135, closes #136

### CODE REVIEW

File: scripts/generate-vendor.sh [hunk 1]
```diff
@@ -227,6 +227,7 @@ extract_blobs() {
       openTag=$(grep '^<?xml version' "$outPath" || true )
       if [[ "$openTag" != "" ]]; then
         grep -v '^<?xml version' "$outPath" > "$TMP_WORK_DIR/xml_fixup.tmp" || true
+        rm -f "$outPath"
```
Comment (anestisb) [inline scripts/generate-vendor.sh]: I don't think this is needed. The following line ensures that the contents of the "$outPath" file is always overwritten.

**Comment by anestisb** [comment]
@cernekee appreciate the PR. There some bits that need to go in the full configs, but I'll iron them out in following commits. Please address the comment and I'll move forward after the merge.

File: scripts/generate-vendor.sh (hunk 1 above)
Comment (cernekee) [reply scripts/generate-vendor.sh]: `echo "$openTag" > "$outPath"` fails if the existing file is mode 0444.

Comment (anestisb) [reply scripts/generate-vendor.sh]: I see. Then maybe it would be a good idea to revert perms after write. Ok, will follow-up with a different CL.

**Pull request merged**